ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -46,6 +46,7 @@ function createMap(insertionElement) {
     var layerCollectionController = new OpenLayerCollectionController({ collection: layerPrefs });
     var map = layerCollectionController.makeMap({
         zoom: 3,
+        minZoom: 1.9,
```

Comment:
✏️ nit: a constant might make more sense here

Reply:
Any reasoning behind that value, by the way?

Reply:
I just picked a value to limit the zoom too. At 1.9 you will get only one partial duplicate of the map instead of four full duplicates.